src/patternfly/layouts/Flex/examples/Flex.md

```diff
@@ -769,6 +769,26 @@ Ordering - Ordering can be applied to nested <code>.pf-l-flex</code> and <code>.
 {{/l-flex}}
 ```
 
+### List type
+```hbs
+{{#> l-flex l-flex--type="ul"}}
+  {{#> wrapper l-flex-item--type="li"}}
```

Comment:
Is there a reason you wouldn't just put `l-flex-item--type="li"` on the `{{#> l-flex}}` block?